mtp_noms_ops/assets-static/images/added-to-my-list.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 5.292 5.292">
```

Comment:
I don't know if it's definitely never going to be reused, but is a more generic name for the SVg (like `tick.svg`) more useful, in case this pattern gets reused?